### PR TITLE
Added AMP Support

### PR DESCRIPTION
**Pull request by milindmore22**
Added AMP Compatability

---
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-wordpress-plugin/276)

### CODE REVIEW

**Comment by milindmore22** [comment]
@rgomezp any updates on this PR? 
ampproject/amp-wp#6453

File: onesignal-public.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ public function __construct()
     public static function init()
     {
         add_action('wp_head', array(__CLASS__, 'onesignal_header'), 10);
+		add_action( 'wp_enqueue_scripts', array( __CLASS__, 'onesigal_amp_style' ) );
```
Comment (rgomezp) [inline onesignal-public.php:24]: Nit: indentation

**Review by rgomezp — CHANGES_REQUESTED** [review]
Howdy @milindmore22 ,
Thanks for your PR and also your patience!

This PR looks great, just left a couple nits for you. Let's get this merged!

Thanks so much again.

File: onesignal-public.php [hunk 2]
```diff
@@ -51,6 +52,20 @@ private static function getOneSignalPluginPath()
 
     public static function onesignal_header()
     {
+		
+		if ( function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
```
[the remark this answers is not in the document]
Comment (milindmore22) [reply onesignal-public.php]: Good catch, replaced with [amp_is_request ](https://amp-wp.org/reference/function/amp_is_request/)

File: onesignal-public.php (hunk 1 above)
Comment (milindmore22) [reply onesignal-public.php]: Updated PR code with space indentation.

**Comment by milindmore22** [comment]
@rgomezp Addressed PR feedback

**Review by rgomezp — APPROVED** [review]

**Review by jkasten2 — COMMENTED** [review]
PR looks good to me

> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-wordpress-plugin/276#-:-MwiCvn38qBV0yh7A4zg:bw5yd0f)* status: all files reviewed, 2 unresolved discussions

**Pull request merged**